app/src/main/res/layout/fragment_new_create_pin.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/contentView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/bg_lime"
+    android:fitsSystemWindows="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <org.p2p.uikit.organisms.UiKitToolbar
+            android:id="@+id/toolbar"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintTop_toTopOf="parent"
+            app:navigationIcon="@drawable/ic_back_night"
+            app:title="@string/auth_create_wallet_step_3"
+            app:titleTextAppearance="@style/WalletTheme.TextAppearance.SemiBold18" />
+
+        <ImageView
+            android:id="@+id/imageViewLock"
```

Comment:
we haven't decided about changing the `type` place in the naming
types should still be at the end of the name